File: PlantView/app/src/main/java/matthewfrost/co/plantview/Location.java
package matthewfrost.co.plantview;

/**
 * Created by Matthew on 21/02/2017.
 */

public class Location
{
    public int ID;
    public String Database;
    public String Table;
    public String Column;
    public String Name;
    public Float Latitude;
    public Float Longitude;
    public String ValueName;

    public Location(int id, String db, String table, String column, String name, Float lat, Float longitude, String VName ){
        ID = id;
        Database = db;
        Table = table;
        Column = column;
        Name = name;
        Latitude = lat;
        Longitude = longitude;
        ValueName = VName;
    }
}

File: PlantView/app/src/main/java/matthewfrost/co/plantview/Location.kt
package matthewfrost.co.plantview

/**
 * Created by Matthew on 14/02/2017.
 */
class Location (name: String, Lat : Float, Long : Float, Val : String){
    public val Name : String = name
    public val Latitude : Float = Lat
    public val Longitude : Float = Long
    public val Value : String = Val
    public var Data : List<Float> = listOf()
}